openapi/paths/credit-memos@{id}.yaml

```
@@ -74,11 +74,11 @@ patch:
     - Core
   tags:
     - Credit memos
-  summary: Update credit memo, allocation, or both
+  summary: Partially update a credit memo
   operationId: PatchCreditMemo
   x-sdk-operation-name: patch
   description: |-
-    Updates a credit memo, credit memo allocation, or both credit memo and allocation with a specified ID.
+    Partially updates a credit memo with specified identifier string.
```

nitpick: I think it reads better with an "a" here
```suggestion
    Partially updates a credit memo with a specified identifier string.
```

---

I couldn't decide between the two! Thanks for the suggestion. :)